pkg/ingester/flush.go

```diff
@@ -135,8 +137,9 @@ func (i *Ingester) sweepStream(instance *instance, stream *stream, immediate boo
 }
 
 func (i *Ingester) flushLoop(j int) {
+	l := log.With(i.logger, "loop", j)
```

Comment:
I added the `loop` to the logger so when a flush fails, or is canceled after max retries, we can see which loop it was.